tests/ValidateSbrpTests/ValidateSbrpTests.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Packaging;
+using NuGet.Packaging.Signing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ValidateSbrpTests;
+
+public class ValidateSbrpTests
+{
+    private string[] Packages { get; set; }
+
+    private const string SbrpAttributeType = "System.Reflection.AssemblyMetadataAttribute";
+
+    private const string VerionPattern = @"(\.\d)+([\-\w])*";
+
+    private string RepoRoot { get; set; }
+
+    public ITestOutputHelper output { get; set; }
+
+    public ValidateSbrpTests(ITestOutputHelper _output)
+    {
+        output = _output;
+
+        RepoRoot = Environment.CurrentDirectory.Substring(0, Environment.CurrentDirectory.IndexOf("artifacts"));
+
+        string buildPackagesDirectory = Path.Combine(RepoRoot, "artifacts/source-build/self/src/artifacts/packages/Release/Shipping");
+
+        if (!Directory.Exists(buildPackagesDirectory))
+        {
+            throw new DirectoryNotFoundException($"Directory {buildPackagesDirectory} does not exist. Try building with './build.sh -sb' first.");
+        }
+
+        Packages = Directory.GetFiles(buildPackagesDirectory, "*.nupkg", SearchOption.AllDirectories);
+    }
+
+    [Fact]
+    public void CheckForSbrpAttribute()
+    {
+        string[] targetPacks = Directory.GetDirectories(Path.Combine(RepoRoot, "src/targetPacks/ILsrc")).Select(x => Path.GetFileName(x).ToLower()).ToArray();
+        string[] textOnlyPackages = Directory.GetDirectories(Path.Combine(RepoRoot, "src/textOnlyPackages/src")).Select(x => Path.GetFileName(x).ToLower()).ToArray();
```

Comment:
Consider breaking these apart into separate lines for readability.

```suggestion
        string[] targetPacks = Directory.GetDirectories(Path.Combine(RepoRoot, "src/targetPacks/ILsrc"))
            .Select(x => Path.GetFileName(x).ToLower())
            .ToArray();
        string[] textOnlyPackages = Directory.GetDirectories(Path.Combine(RepoRoot, "src/textOnlyPackages/src"))
            .Select(x => Path.GetFileName(x).ToLower())
            .ToArray();
```

Reply:
Changed in https://github.com/dotnet/source-build-reference-packages/commit/ce1a70dc071c6a5adce2b2d0ca6285031ee12258